src/main/resources/api/swagger.yaml

```diff
@@ -89,27 +89,27 @@ paths:
           schema:
             $ref: '#/definitions/Error'
     post:
-      summary: Creates a new feature.
-      description: Creates a new feature with specified name, docker data, etc. Feature name is checked to be unique.
-      tags:
-        - Features
-      parameters:
-        - name: feature
-          required: true
-          in: body
-          schema:
-            $ref: '#/definitions/Feature'
-      consumes:
-        - application/json
-      responses:
-        201:
-          description: Feature successfully created.
-          schema:
-            $ref: '#/definitions/Feature'
-        default:
-          description: Unexpected error.
-          schema:
-            $ref: '#/definitions/Error'
+        summary: Creates a new feature.
+        description: Creates a new feature with specified name, docker data, etc. Feature name is checked to be unique. status, author and created_at must not be filled.
```

Comment:
Must or should? Must gives the impression that the API will break if the user sends these fields... or is it just my impression?